### PR TITLE
Plp grid layout

### PR DESCRIPTION
**Pull request by IvansZuks**

### CODE REVIEW

**Comment by ejnshtein** [comment]
Should we add some borders or smth for categories? If they're empty, it looks overall empty.
Screen Shot 2021-03-08 at 12 16 39

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js [hunk 1]
```diff
@@ -68,16 +70,40 @@ export class CategoryPage extends PureComponent {
         isMatchingInfoFilter: false,
         totalPages: 1,
         is_anchor: true,
-        search: ''
+        search: '',
+        defaultPlpType: '',
+        plpTypes: []
     };
 
+    state = {};
```
Comment (ejnshtein) [inline packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js:78]: Maybe set default state here? You are setting it in the `componentDidUpdate` method which is called basically right away.

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.container.js [hunk 2]
```diff
@@ -557,6 +579,20 @@ export class CategoryPageContainer extends PureComponent {
         });
     }
 
+    updatePlpType() {
+        const { plpType } = this.props;
+
+        if (plpType.match('-')) {
+            const plpTypes = plpType.split('-');
+
+            this.setState({ defaultPlpType: plpTypes[0] });
```
Comment (ejnshtein) [inline packages/scandipwa/src/route/CategoryPage/CategoryPage.container.js:588]: You can set multiple state values in a single call of `setState()`.

**Review by ejnshtein — CHANGES_REQUESTED** [review]
Answer to my comments. Thanks!

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js (hunk 1 above)
Comment (IvansZuks) [reply packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js:78]: How can you put value from prop to state here?

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.container.js (hunk 2 above)
Comment (IvansZuks) [reply packages/scandipwa/src/route/CategoryPage/CategoryPage.container.js]: Fixed

**Comment by IvansZuks** [comment]
> Should we add some borders or smth for categories? If they're empty, it looks overall empty.
> Screen Shot 2021-03-08 at 12 16 39

How it is connected with this PR?

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js (hunk 1 above)
Comment (ejnshtein) [reply packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js:78]: `state={key:value}`?

**Comment by ejnshtein** [comment]
> > Should we add some borders or smth for categories? If they're empty, it looks overall empty.
> > Screen Shot 2021-03-08 at 12 16 39
> 
> How it is connected with this PR?

Looks wrong to me. Just wanted to know if it's ok.

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.container.js (hunk 2 above)
Comment (ejnshtein) [reply packages/scandipwa/src/route/CategoryPage/CategoryPage.container.js]: Nice, thanks!

File: packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js (hunk 1 above)
Comment (IvansZuks) [reply packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js:78]: I mean, how to get value from props here correctly

Comment (ejnshtein) [reply packages/scandipwa/src/route/CategoryPage/CategoryPage.component.js:78]: Of right, there's a prop value. Sorry, didn't notice it...
If that's the case I don't see an issue.

**Review by ejnshtein — APPROVED** [review]

**Pull request merged**